frontend/src/metabase/common/components/Alert/Alert.tsx

```diff
@@ -29,7 +31,18 @@ const Alert = ({
       role="alert"
     >
       {icon && <AlertIcon variant={variant} name={icon} size={24} />}
+
       <div>{children}</div>
+
+      {onClose && (
+        <CloseIcon
+          variant="info"
+          name="close"
+          size={24}
+          onClick={onClose}
+          data-testid="alert-close-button"
+        />
+      )}
```

Comment:
We need a close icon in the Hosted Bundle PR for alert

Reply:
You might want to ping design system folks as we're adding to the design system